docs/resources/vrf.md

```diff
@@ -7,81 +10,439 @@ description: |-
-
-- `relation_fv_rs_ospf_ctx_pol` - (Optional) Relation to class ospfCtxPol. Cardinality - N_TO_ONE. Type - String.
-- `relation_fv_rs_vrf_validation_pol` - (Optional) Relation to class l3extVrfValidationPol. Cardinality - N_TO_ONE. Type - String.
-- `relation_fv_rs_ctx_mcast_to` - (Optional) Relation to class vzFilter. Cardinality - N_TO_M. Type - [Set of String].
-- `relation_fv_rs_ctx_to_eigrp_ctx_af_pol` - (Optional) Relation to class eigrpCtxAfPol. Cardinality - N_TO_M. Type - [Set of Map].
-- `relation_fv_rs_ctx_to_ospf_ctx_pol` - (Optional) Relation to class ospfCtxPol. Cardinality - N_TO_M. Type - [Set of Map].
-- `relation_fv_rs_ctx_to_ep_ret` - (Optional) Relation to class fvEpRetPol. Cardinality - N_TO_ONE. Type - String.
-- `relation_fv_rs_bgp_ctx_pol` - (Optional) Relation to class bgpCtxPol. Cardinality - N_TO_ONE. Type - String.
-- `relation_fv_rs_ctx_mon_pol` - (Optional) Relation to class monEPGPol. Cardinality - N_TO_ONE. Type - String.
-- `relation_fv_rs_ctx_to_ext_route_tag_pol` - (Optional) Relation to class l3extRouteTagPol. Cardinality - N_TO_ONE. Type - String.
-- `relation_fv_rs_ctx_to_bgp_ctx_af_pol` - (Optional) A block representing the relationship to class bgpCtxAfPol. Cardinality - N_TO_M. Type - Block.
-  * `af` - (Required) The BGP address family. Allowed values are "ipv4-ucast", "ipv6-ucast", and default value is "ipv4-ucast". Type: String.
-  * `tn_bgp_ctx_af_pol_name` - (Required) Distinguished name (DN) of the BGP address family context policy object. Type: String.
-
-
-Note: In the APIC GUI,a VRF (fvCtx) was called a "Context"or "PrivateNetwork."
-
-## Attribute Reference ##
-
-The only attribute that this resource exports is the `id`, which is set to the
-Dn of the VRF.
+All examples for the VRF resource can be found in the [examples](https://github.com/CiscoDevNet/terraform-provider-aci/tree/master/examples/resources/aci_vrf) folder.
+
+## Schema ##
+
+### Required ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_tenant](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/tenant) ([fvTenant](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvTenant/overview))
+* `name` (name) - (string) The name of the VRF object.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the VRF object.
+
+### Optional ###
+
+* `annotation` (annotation) - (string) The annotation of the VRF object.
+  - Default: `orchestrator:terraform`
+* `bd_enforcement` (bdEnforcedEnable) - (string) The Bridge Domain (BD) enforcement status of the VRF. BD enforcement will restrict a host from sending ping requests to subnets outside the Bridge Domain (BD). The feature blocks ICMP, TCP and UDP traffic between BDs.
+  - Default: `false`
+  - Valid Values: `no`, `yes`.
```

Comment:
```suggestion
  - Default: `no`
  - Valid Values: `no`, `yes`.
```
`false` doesn't appear to be a valid value.

Reply:
changed